smartcheck/check.py

```diff
@@ -116,6 +146,10 @@ def __init__(self, file_or_string, db_path=None):
         self._database = None
         self._generic = None
 
+        # boolean field, either true: is nvme, or false: not an nvme
+        self.is_nvme = None
```

Comment:
Can be simplified to 

```
self.is_nvme = DATA_SECTION_START_NVME in raw self.raw and DATA_SECTION_START not in self.raw
if self.is_nvme:
    self.db_path = DEFAULT_DISKS_FILE_NVME
```

which eliminates `set_nvme`

Reply:
Removed `set_nvme` and used your suggestion.